spec/models/dish_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Dish, type: :model do
+  it 'has a valid factory' do
+    expect(FactoryGirl.create(:dish)).to be_valid
```

Comment:
I thought you guys included the FactoryGirl methods so you could use the short syntax.